### DISCUSSION TITLE
FEATURE REQUEST : Pull run name of different workflows dynamically in a drop down menu

### DISCUSSION THREAD

**Discussion by giet12aei033** [community/community#142818] [Actions]
### Select Topic Area

General

### Body

It would be beneficial to have a GitHub Action that retrieves the run names of different workflows and presents them in a dropdown menu. This feature would assist in the deployment workflow, which involves multiple build jobs, allowing us to consolidate everything into one main Helm chart for deployment.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @giet12aei033 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Actions category. I've gone ahead and moved it for you. Good luck!
